controllers/ssp_controller.go

```diff
@@ -596,31 +608,33 @@ func watchSspResource(bldr *ctrl.Builder) {
 	bldr.For(&ssp.SSP{}, builder.WithPredicates(pred))
 }
 
-func watchNamespacedResources(builder *ctrl.Builder) {
+func watchNamespacedResources(builder *ctrl.Builder, opers []operands.Operand) {
```

Comment:
wouldn't be better to use better name than `opers`? e.g. sspOperands

Reply:
yes, I will change it.